include/velocypack/Serializable.h

```diff
@@ -47,6 +48,18 @@ struct Serialize {
   Serializable const& _sable;
 };
 
+class LambdaSerializable : public Serializable {
+ public:
+  std::function<void(Builder&)> lambda;
+
+  LambdaSerializable(std::function<void(Builder&)> lambda) : lambda(lambda) {}
+
+  using Serializable::toVelocyPack;
+  void toVelocyPack(Builder& b) const override {
+    lambda(b);
+  }
+};
```

Comment:
The use of `std::function` seems unnecessary here because the lambda type is known. I suggest this (untested) alternative:
```suggestion
template<typename F>
struct LambdaSerializable : Serializable, F {
    static_assert(std::is_invocable_r_v<void, F, Builder&>);
    explicit LambdaSerializable(F f) : F(std::move(f)) {}
    void toVelocyPack(Builder& b) const final {
        this->operator()(b);
    }
};
```

Reply:
Works, except for the std::is_invocable_r_v that is too recent.